web/src/lib/utils/asset-utils.ts

```diff
@@ -224,6 +227,44 @@ export const getSelectedAssets = (assets: Set<AssetResponseDto>, user: UserRespo
   return ids;
 };
 
+export const selectAllAssets = async (assetStore: AssetStore, assetInteractionStore: AssetInteractionStore) => {
+  if (get(isSelectingAllAssets)) {
+    // Selection is already ongoing
+    return;
+  }
+  isSelectingAllAssets.set(true);
+
+  try {
+    let selectedAssetCount = 1;
+    loadBuckets: for (const bucket of get(assetStore).buckets) {
```

Comment:
Using `get` is unnecessary

Reply:
Isn't this a subscription? And this isn't a svelte component.

Reply:
AssetStore is subscribable, but since we already have the value, using get is unnecessary